core/checks/count_check.py

```
@@ -0,0 +1,158 @@
+        # Set TTL on first increment (when count == 1)
+        if current_count == 1:
+            ttl_seconds = 86400  # 24 hours
+            if get_setting_func:
+                ttl_str = await get_setting_func('onboarding_ttl_seconds', default='86400')
+                try:
+                    ttl_seconds = int(ttl_str)
+                except ValueError:
+                    ttl_seconds = 86400
+            
+            await redis_client.expire(count_key, ttl_seconds)
+            logger.info(f"Set {count_key} TTL to {ttl_seconds}s")
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)